test/components/tableTests.ts

```diff
@@ -1,288 +1,112 @@
-  });
+        let table = new Plottable.Components.Table();
+        table.add(c00, 0, 0);
+        table.add(c11, 1, 1);
 
-  it("table.requestedSpace works properly", () => {
-    let c0 = new Plottable.Component();
-    let c1 = TestMethods.makeFixedSizeComponent(50, 50);
-    let c2 = TestMethods.makeFixedSizeComponent(20, 50);
-    let c3 = TestMethods.makeFixedSizeComponent(20, 20);
-
-    let table = new Plottable.Components.Table([
-      [c0, c1],
-      [c2, c3]
-    ]);
-
-    let spaceRequest = table.requestedSpace(30, 30);
-    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "1");
-
-    spaceRequest = table.requestedSpace(50, 50);
-    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "2");
-
-    spaceRequest = table.requestedSpace(90, 90);
-    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "3");
-
-    spaceRequest = table.requestedSpace(200, 200);
-    TestMethods.verifySpaceRequest(spaceRequest, 70, 100, "4");
+        assert.isNull(table.componentAt(0, 1), "returns null if an empty cell is queried");
+        assert.isNull(table.componentAt(-1, 0), "returns null if a negative row index is passed in");
+        assert.isNull(table.componentAt(0, -1), "returns null if a negative column index is passed in");
+        assert.isNull(table.componentAt(9001, 0), "returns null if a row index larger than the number of rows is passed in");
+        assert.isNull(table.componentAt(0, 9001), "returns null if a column index larger than the number of columns is passed in");
+      });
+    });
   });
 
-  describe("table._iterateLayout works properly", () => {
-    // This test battery would have caught #405
-    function verifyLayoutResult(result: any,
-                                cPS: number[], rPS: number[], gW: number[], gH: number[],
-                                wW: boolean, wH: boolean, id: string) {
-      assert.deepEqual(result.colProportionalSpace, cPS, "colProportionalSpace:" + id);
-      assert.deepEqual(result.rowProportionalSpace, rPS, "rowProportionalSpace:" + id);
-      assert.deepEqual(result.guaranteedWidths, gW, "guaranteedWidths:" + id);
-      assert.deepEqual(result.guaranteedHeights, gH, "guaranteedHeights:" + id);
-      assert.deepEqual(result.wantsWidth, wW, "wantsWidth:" + id);
-      assert.deepEqual(result.wantsHeight, wH, "wantsHeight:" + id);
-    }
-
-    it("iterateLayout works in the easy case where there is plenty of space and everything is satisfied on first go", () => {
-      let c1 = new Mocks.FixedSizeComponent(50, 50);
-      let c2 = new Plottable.Component();
-      let c3 = new Plottable.Component();
-      let c4 = new Mocks.FixedSizeComponent(20, 10);
-      let table = new Plottable.Components.Table([
-        [c1, c2],
-        [c3, c4]
-      ]);
-      let result = (<any> table)._iterateLayout(500, 500);
-      verifyLayoutResult(result, [215, 215], [220, 220], [50, 20], [50, 10], false, false, "");
+  describe("add()", () => {
```

Comment:
I was thinking I should do that, but I'm not sure how to name this `describe()` block then. Do I just do so without calling out the name of the method itself?